base/string_utils.cpp

```diff
@@ -113,6 +113,21 @@ UniString Normalize(UniString const & s)
   return result;
 }
 
+void NormalizeDigits(string & utf8)
+{
+  for (size_t i = 0; i + 2 < utf8.size(); ++i)
+  {
+    if (utf8[i] == '\xEF' && utf8[i + 1] == '\xBC')
+    {
+      char const n = utf8[i + 2];
+      if (n < '\x90' || n > '\x99')
+        continue;
+      utf8[i] = n - 0x90 + '0';
+      utf8.erase(i + 1, 2);
```

Comment:
?

> On Apr 1, 2016, at 21:18, ygorshenin <notifications@github.com> wrote:
> 
> In base/string_utils.cpp <https://github.com/mapsme/omim/pull/2670#discussion_r58245888>:
> 
> > @@ -113,6 +113,21 @@ UniString Normalize(UniString const & s)
> >    return result;
> >  }
> >  
> > +void NormalizeDigits(string & utf8)
> > +{
> > +  for (size_t i = 0; i + 2 < utf8.size(); ++i)
> > +  {
> > +    if (utf8[i] == '\xEF' && utf8[i + 1] == '\xBC')
> > +    {
> > +      char const n = utf8[i + 2];
> > +      if (n < '\x90' || n > '\x99')
> > +        continue;
> > +      utf8[i] = n - 0x90 + '0';
> > +      utf8.erase(i + 1, 2);
> Нет, за квадрат не Айс. Переделал за линию.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2670/files/af10e69d522742081c4007f6da63a022705f438b#r58245888>